### DISCUSSION TITLE
Option to favorite currencies and default currency

### DISCUSSION THREAD

**Discussion by Infiniti151** [ferraridamiano/ConverterNOW#312] [Ideas]
Currency conversion works great, but I have to scroll through the long list of currencies to find the two currencies. It would be great if we could favorite currencies, so that those would always be at the top. Another option for default currency would also be useful, so that the default one is auto-selected when currency converter opens. So all that would be required is type numbers instead of unnecessary clicks and scrolls.

**Comment by ferraridamiano** [comment]
Hello @aadish151

What is the point of favorite units when you can reorder them?

**Reply by Infiniti151** [reply]
Thanks. I didn't see that option. That solves my issue.
